docs/asf-docs/Introduction_hyp3_mintpy.ipynb

```diff
@@ -0,0 +1,527 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!conda install -y -c conda-forge hyp3_sdk"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os, sys\n",
+    "import re\n",
+    "import glob\n",
+    "import zipfile\n",
+    "import json\n",
+    "from typing import no_type_check\n",
+    "from datetime import datetime, timezone\n",
+    "import numpy as np\n",
+    "from argparse import ArgumentParser\n",
+    "import hyp3_sdk as sdk\n",
+    "from hyp3_sdk import asf_search"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2.1. Download the hyp3 data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set the home directory, project name, output directory, and granule_list file. The granule_list file includes the RTC file names that users wants to use them to produce the interferograms for the time-series analysis purpose. Users\n",
+    "usually provide this file preapred with ASF vertex (https://serach.asf.alaska.edu)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hyp3_mintpy_home = \"/home/jzhu4/projects/work/hyp3/hyp3-mintpy\"\n",
+    "projectname = \"case5\"\n",
+    "outdir = \"/media/jzhu4/data/hyp3-mintpy/case5\"\n",
+    "granule_list = \"/media/jzhu4/data/hyp3-mintpy/case5/granule.dat\"\n",
+    "os.chdir(outdir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "User autentication"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hyp3 = sdk.HyP3(prompt = True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Submit the jobs with hyp3-sdk and download the files "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def run_job(hyp3, granule_list_file, projectname, outdir):\n",
+    "    #read the granulelist file\n",
+    "    with open(granule_list_file) as f:\n",
+    "        lines = f.read().splitlines()\n",
+    "\n",
+    "    num = len(lines)\n",
+    "    reference_id = lines[num-1]  \n",
+    "    insar_jobs = sdk.Batch()\n",
+    "    \n",
+    "    for i in range(num-2, 0, -1):\n",
+    "        secondary_id = lines[i]\n",
+    "        job_name = projectname\n",
+    "        insar_jobs += hyp3.submit_insar_job(reference_id, secondary_id, job_name,\n",
+    "                                    include_look_vectors=True, include_inc_map=True,\n",
+    "                                    include_los_displacement=True, include_dem=True,\n",
+    "                                    include_wrapped_phase=True)\n",
```

Comment:
I think the dem is the only optional file that MintPy currently relies on, I'd recommend not including the others to minimize the size of the files that the user will have to download:
```suggestion
    "        insar_jobs += hyp3.submit_insar_job(reference_id, secondary_id, job_name, include_dem=True)\n",
```

Reply:
changed